walletconnect/package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "vanillajs-walletconnect",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview"
+  },
+  "devDependencies": {
+    "vite": "^5.4.0"
```

prefer exact version numbers for dependencies.

```suggestion
    "vite": "5.4.0"
```